esmvaltool/cmorizers/data/downloaders/datasets/cds_xch4.py

```diff
@@ -0,0 +1,39 @@
+"""Script to download CDS-XCH4 from the Climate Data Store (CDS)."""
```

Comment:
The downloader works fine but there is a naming issue with the current cmorizer framewrok. This script downloads a file named: 200301_201812-C3S-L3_GHG-GHG_PRODUCTS-MERGED-MERGED-OBS4MIPS-MERGED-v4.1.nc but the formatter script assumes that the raw file is named: OBS_CDS-XCH4_sat_L3_xch4_200301-201612.nc
Thus if you use the downloaders and the formatter separetely, this fails. If you do both at once (prepare), it works fine. Could you fix the filename here or in the formatter script?